lib/network/modules/components/Edge.js

```diff
@@ -256,7 +256,7 @@ class Edge {
       parentOptions.selfReference = {
           size: newOptions.selfReferenceSize,
           angle: parentOptions.selfReference.angle,
-          fullRender: parentOptions.selfReference.fullRender
+          renderBehindTheNode: parentOptions.selfReference.renderBehindTheNode
       }
```

Comment:
Can't you just change the `size` here instead of creating a new object? I mean if `selfReference` isn't an object it will crash anyway so what's the difference? Can't it crash btw?